demos/src/icons.mustache

```diff
@@ -1,35 +1,34 @@
 <h2>Currently supported icons</h2>
-<p>You can add any icon using the <code>oButtonsIconButton</code> mixin. If you need an icon button with the build service, please ask the team or raise a PR adding the o-icons icon to the <code>$o-buttons-icons</code> variable.</p>
 	{{#icons}}
-		<button class="o-buttons o-buttons-icon o-buttons-icon--{{name}}">
+		<button class="o-buttons o-buttons--secondary o-buttons-icon o-buttons-icon--{{name}}">
```

Comment:
i'm noticing some of the mustache is indented with tabs and some with spaces, not really a thing for this PR but i'm going to add to my notebook that `oat` should have some linting (and fixing) for demo templates